datastore/emulator.go

```diff
@@ -104,3 +116,31 @@ func Emulator(ctx context.Context) <-chan struct{} {
 
 	return quit
 }
+
+func findEmulatorPath() string {
+	gcloudPath, err := exec.LookPath("gcloud")
```

Comment:
Go style encourages short variable names for small scopes. `path` or even `p` would be preferable here.

Reply:
Done.